README.md

```diff
@@ -117,12 +154,16 @@ Create a new cluster
 $ k8t new cluster MyCluster
 ```
 
-Create a new environment
+Create a new global environment
 
 ```bash
-$ k8t new environment staging
+$ k8t new environment staging -c MyCluster
 ```
 
+And a new cluster environment
+
+```bash
+k8t new environment staging -c MyCluster
 Generate a new deployment template for cluster MyCluster (for a list of available templates see the `k8t new template --help`)
 
```

Comment:
can't really comment this with a suggestion, but the inline code snippets are broken